internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,39 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
 var (
-	StepLength = 0.65 // длина шага в метрах
+	StepLength = 0.65 // длина шага в метрах.
 )
 
+// parsePackage: Мы передаем в функцию строку, в которой всегда должно быть кол-во шагов
+// и продолжительность прогулки. Если одного из параметров нет - возвращаем ошибку. На выход функция
+// выдает кол-во шагов и продолжительность прогулки в формате int и time, т.е. парсит.
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	// Делим входящую строку на части, делаем слайсы.
+	separatePart := strings.Split(data, ",")
+	if len(separatePart) != 2 {
+		return 0, 0, errors.New("Ошибка: к нам не пришло одно из значений")
+	}
+	// Преобразуем строку в число, возвращаем int кол-ва шагов.
+	countSteps, err := strconv.Atoi(separatePart[0])
+	if err != nil {
+		return 0, 0, errors.New("Ошибка: неверный формат данных для шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
Привет! Спасибо огромное, принято, я потом перепишу по ошибкам, забрал себе)